README.md

```diff
@@ -172,3 +172,8 @@ Concurrency
 -----------
 
 Thimble uses the [throng](https://www.npmjs.com/package/throng) module to leverage Node's [Cluster API](https://nodejs.org/api/cluster.html) for concurrency. To specify the number of server processes to start set `WEB_CONCURRENCY` to a positive integer value.
+
+Chat with the team
+-----------
+
+You can talk with the team in the [contributing channel]( http://chat.mozillafoundation.org) on [mattermost](https://about.mattermost.com). We're a friendly group! 
```

Comment:
I think we should also change the text to something like:

We're a friendly group, so feel free to chat with us in the "Thimble" channel on Mozilla Chat running on [Mattermost](https://about.mattermost.com). To access Mozilla Chat head to [this link](https://chat.mozillafoundation.org). Note that you will need to create an account first.

You can also download a mobile or desktop client for Mattermost [here](https://about.mattermost.com/download/#mattermostApps).